R/zchunk_L201.en_nonco2.R

```diff
@@ -0,0 +1,358 @@
+      steepness <- SO2 <- NOx <- CO <- BC <- OC <- NMVOC <- depresource <- has_district_heat <-
+      . <- region <- supplysector <- NULL  # silence package check notes
+
+    # Load required inputs
+    GCAM_region_names <- get_data(all_data, "common/GCAM_region_names")
+    A_regions <- get_data(all_data, "emissions/A_regions")
+    A_regions.en <- get_data(all_data, "energy/A_regions")
+
+    L111.nonghg_tg_R_en_S_F_Yh <- get_data(all_data, "temp-data-inject/L111.nonghg_tg_R_en_S_F_Yh") %>%
+      # temporary for temp-data-inject
+      gather(year, value, -1:-5) %>% mutate(year = as.integer(substr(year, 2, 5)))
+    L111.nonghg_tgej_R_en_S_F_Yh <- get_data(all_data, "temp-data-inject/L111.nonghg_tgej_R_en_S_F_Yh") %>%
+      # temporary for temp-data-inject
+      gather(year, value, -1:-5) %>% mutate(year = as.integer(substr(year, 2, 5)))
+    L112.ghg_tg_R_en_S_F_Yh <- get_data(all_data, "temp-data-inject/L112.ghg_tg_R_en_S_F_Yh") %>%
+      # temporary for temp-data-inject
+      gather(year, value, -1:-5) %>% mutate(year = as.integer(substr(year, 2, 5)))
+    L112.ghg_tgej_R_en_S_F_Yh <- get_data(all_data, "temp-data-inject/L112.ghg_tgej_R_en_S_F_Yh") %>%
+      # temporary for temp-data-inject
+      gather(year, value, -1:-5) %>% mutate(year = as.integer(substr(year, 2, 5)))
+
+    L114.bcoc_tgej_R_en_S_F_2000 <- get_data(all_data, "L114.bcoc_tgej_R_en_S_F_2000")
+    L151.nonghg_ctrl_R_en_S_T <- get_data(all_data, "L151.nonghg_ctrl_R_en_S_T")
+    A51.steepness <- get_data(all_data, "emissions/A51.steepness")
+    L244.DeleteThermalService <- get_data(all_data, "temp-data-inject/L244.DeleteThermalService")
+
+    # L201.en_pol_emissions: Pollutant emissions for energy technologies in all regions
+    L111.nonghg_tg_R_en_S_F_Yh %>%
+      filter(supplysector != "out_resources",
+             year %in% emissions.MODEL_BASE_YEARS) %>%
+      # add region name and round output
+      left_join_error_no_match(GCAM_region_names, by = "GCAM_region_ID") %>%
+      select(region, supplysector, subsector, stub.technology, year, input.emissions = value, Non.CO2) %>%
+      mutate(input.emissions = round(input.emissions, emissions.DIGITS_EMISSIONS)) ->
+      L201.en_pol_emissions
+
+    # L201.en_ghg_emissions: GHG emissions for energy technologies in all regions
+    L112.ghg_tg_R_en_S_F_Yh %>%
+      filter(supplysector != "out_resources",
+             year %in% emissions.MODEL_BASE_YEARS) %>%
+      # add region name and round output
+      left_join_error_no_match(GCAM_region_names, by = "GCAM_region_ID") %>%
+      select(region, supplysector, subsector, stub.technology, year, input.emissions = value, Non.CO2) %>%
+      mutate(input.emissions = round(input.emissions, emissions.DIGITS_EMISSIONS)) ->
+      L201.en_ghg_emissions
+
+    # L201.en_bcoc_emissions: BC/OC emissions factors for energy technologies in all regions
+    # Interpolate and add region name
```

Comment:
I'm going to guess that is a comment from the old data system because that was the name of the function called. I don't think it actually interpolates...we only had one year of BC/OC data.